============================= test session starts ==============================
platform linux -- Python 3.10.6, pytest-7.3.1, pluggy-1.0.0
rootdir: /data/sandeep/leetcode-master/.verifaiws
collected 2 items

538_Convert_BST_to_Greater_Tree/test_538_Convert_BST_to_Greater_Tree.py F [ 50%]
.                                                                        [100%]

=================================== FAILURES ===================================
_________ test_convert_bst_should_return_the_root_with_correct_values __________

solution = <solution.Solution object at 0x7fd0ec1d8af0>
binary_search_tree = <test_538_Convert_BST_to_Greater_Tree.binary_search_tree.<locals>.BinaryTreeNode object at 0x7fd0ec1d8c10>

    def test_convert_bst_should_return_the_root_with_correct_values(solution, binary_search_tree):
        result_root = solution.convertBST(binary_search_tree)
    
        assert result_root.val == 22
>       assert result_root.left.val == 29
E       assert 28 == 29
E        +  where 28 = <test_538_Convert_BST_to_Greater_Tree.binary_search_tree.<locals>.BinaryTreeNode object at 0x7fd0ec1d8c70>.val
E        +    where <test_538_Convert_BST_to_Greater_Tree.binary_search_tree.<locals>.BinaryTreeNode object at 0x7fd0ec1d8c70> = <test_538_Convert_BST_to_Greater_Tree.binary_search_tree.<locals>.BinaryTreeNode object at 0x7fd0ec1d8c10>.left

538_Convert_BST_to_Greater_Tree/test_538_Convert_BST_to_Greater_Tree.py:43: AssertionError
=========================== short test summary info ============================
FAILED 538_Convert_BST_to_Greater_Tree/test_538_Convert_BST_to_Greater_Tree.py::test_convert_bst_should_return_the_root_with_correct_values
========================= 1 failed, 1 passed in 0.01s ==========================
